### PR TITLE
채팅방 삭제하기 기능 구현

### PR DESCRIPTION
**Pull request by naekang**
- [ ] 채팅방 삭제하기 기능 구현
- [ ] 테스트 코드
- [ ] Spring Rest Docs

### CODE REVIEW

**Comment by naekang** [comment]
|File|Coverage [85.52%]|:green_apple:|
|:-|:-:|:-:|
|[ChatServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fservice%2FChatServiceImpl.java)|100%|:green_apple:|
|[ChatController.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fcontroller%2FChatController.java)|100%|:green_apple:|
|[ChatErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fexception%2Ferrorcode%2FChatErrorCode.java)|100%|:green_apple:|
|[ChatRoom.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatRoom.java)|69.12%|:x:|

|Total Project Coverage|65.04%|:x:|
|:-|:-:|:-:|

**Comment by naekang** [comment]
|File|Coverage [85.52%]|:green_apple:|
|:-|:-:|:-:|
|[ChatServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fservice%2FChatServiceImpl.java)|100%|:green_apple:|
|[ChatController.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fcontroller%2FChatController.java)|100%|:green_apple:|
|[ChatErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fexception%2Ferrorcode%2FChatErrorCode.java)|100%|:green_apple:|
|[ChatRoom.java](https://github.com/Gaethering/Gaethering-Server/blob/0a5cd8165d7d8cfa0a321be37551c15f803e1901/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatRoom.java)|69.12%|:x:|

|Total Project Coverage|65.04%|:x:|
|:-|:-:|:-:|

File: src/main/java/com/gaethering/gaetheringserver/domain/chat/repository/WalkingTimeRepository.java [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface WalkingTimeRepository extends JpaRepository<WalkingTime, Long> {
 
+    @Modifying
+    @Query("delete from WalkingTime w where w.chatRoom = :chatRoom")
+    void deleteAllByChatRoomId(@Param("chatRoom") ChatRoom chatRoom);
```
Comment (younghoondoodoom) [inline src/main/java/com/gaethering/gaetheringserver/domain/chat/repository/WalkingTimeRepository.java:14]: 혹시 deleteAllByChatRoom으로는 안되나요??

**Review by younghoondoodoom — COMMENTED** [review]
고생하셨습니다!

**Comment by naekang** [comment]
|File|Coverage [85.52%]|:green_apple:|
|:-|:-:|:-:|
|[ChatServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/329780733c678d7ea49814550a45524551497298/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fservice%2FChatServiceImpl.java)|100%|:green_apple:|
|[ChatController.java](https://github.com/Gaethering/Gaethering-Server/blob/329780733c678d7ea49814550a45524551497298/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fcontroller%2FChatController.java)|100%|:green_apple:|
|[ChatErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/329780733c678d7ea49814550a45524551497298/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fexception%2Ferrorcode%2FChatErrorCode.java)|100%|:green_apple:|
|[ChatRoom.java](https://github.com/Gaethering/Gaethering-Server/blob/329780733c678d7ea49814550a45524551497298/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatRoom.java)|69.12%|:x:|

|Total Project Coverage|65.04%|:x:|
|:-|:-:|:-:|

**Pull request merged**